packages/javascript-sdk/tsconfig.lib.json

```diff
@@ -4,13 +4,8 @@
     "outDir": "../../dist/packages/javascript-sdk",
     "declaration": true,
     "lib": ["ES2015", "dom"],
-    "moduleResolution": "node",
-    "noFallthroughCasesInSwitch": true,
-    "noImplicitAny": false,
-    "noImplicitReturns": true,
-    "noImplicitThis": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
+    "moduleResolution": "Node",
+    "composite": true,
```

Comment:
I moved stuff that was reused in multiple tsconfigs to the base config, trying to clean up each level of the config that we could.

Reply:
remember this is the ts config that rollup uses, the cjs / esm tsconfigs are not connected